Makefile

```diff
@@ -30,17 +49,34 @@ ifndef GITHUB_AUTH_TOKEN
 endif
 
 .PHONY: e2e
+e2e:  ## Runs e2e tests
 # export GITHUB_AUTH_TOKEN with personal access token to run the e2e
 e2e: build check-env ginkgo
 	$(GINKGO) --skip="E2E TEST:executable" -p -v -cover  ./...
 
 ginkgo:
 	GO111MODULE=off go get -u github.com/onsi/ginkgo/ginkgo
+
 .PHONY: ci-e2e
+ci-e2e:  ## Runs ci e2e tests
 # export GITHUB_AUTH_TOKEN with personal access token to run the e2e
 ci-e2e: build check-env
 	$(call ndef, GITHUB_AUTH_TOKEN)
 	mkdir -p bin
 	./scorecard --repo=https://github.com/ossf/scorecard --format json > ./bin/results.json
 	ginkgo -p  -v -cover  ./...
 
+
+.PHONY: go-mod
+go-mod: ## Cleanup and verify go modules
```

Comment:
nit: Don't see the need of a seperate go-mod target, can just inline into verify-go-mod